client/my-sites/customer-home/cards/features/launch-site/index.jsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { Card, Button } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import CardHeading from 'components/card-heading';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import isAtomicSite from 'state/selectors/is-site-automated-transfer';
+import isSiteChecklistComplete from 'state/selectors/is-site-checklist-complete';
+import { launchSiteOrRedirectToLaunchSignupFlow } from 'state/sites/launch/actions';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export const LaunchSite = ( { isAtomic, isChecklistComplete, launchSite, siteId } ) => {
+	const translate = useTranslate();
+	const isPrimary = ! isAtomic && isChecklistComplete;
+	const onLaunchBannerClick = e => {
+		e.preventDefault();
```

Comment:
We can probably drop this, though I know it existed before

Reply:
b8139505baf1b545d89a4926233a2942eeef8bdb